evo_researcher/autonolas/research.py

```diff
@@ -1076,11 +1076,11 @@ def fetch_additional_information(
     research_chain = (
         research_prompt |
         ChatOpenAI(
-            model_name=engine,
+            model=engine,
```

Comment:
After locking dependencies with PMAT, mypy complained about these. So I just renamed it.